### ISSUE TITLE
trojan virus

### ISSUE THREAD

**Issue by BootlegTheMagician** [cookgreen/BF1BotMod#4]
my antivirus labels it as a trojan virus.

**Comment by rasmus-09** [comment]
no, it's not a virus. I personally started (tried) the injector, but I can’t figure out one problem. Have you been able to play with this mod?

**Comment by BootlegTheMagician** [comment]
i havent as i was afraid this thing was a virus, im going to reinstall it now
